crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -1239,13 +1239,7 @@ impl PgIndexerStore {
             for transaction_chunk in transactions.chunks(PG_COMMIT_CHUNK_SIZE) {
                 diesel::insert_into(transactions::table)
                     .values(transaction_chunk)
-                    .on_conflict(transactions::transaction_digest)
-                    .do_update()
-                    .set((
-                        transactions::timestamp_ms.eq(excluded(transactions::timestamp_ms)),
-                        transactions::checkpoint_sequence_number
-                            .eq(excluded(transactions::checkpoint_sequence_number)),
-                    ))
+                    .on_conflict_do_nothing()
```

Comment:
codes before were for fast path, where the fast path will commit first, and later the checkpoint will commit later and update here.
no longer needed b/c we will deprecate fast path.